great_expectations/profile/basic_dataset_profiler.py

```diff
@@ -96,7 +103,7 @@ def _profile(cls, dataset):
                     df.expect_column_values_to_be_unique(column)
                 elif cardinality in ["one", "two", "very few", "few"]:
                     df.expect_column_distinct_values_to_be_in_set(column, value_set=None, result_format="SUMMARY")
-                else:
+                elif cardinality in ["many", "very many", "unique"]:
```

Comment:
Call these expectations only for known cardinalities - not doing this creates a cascade of errors when cardinality computation failed.

Reply:
:+1:

Reply:
I think it is useful to add a log output at least in the event that we find via profiling some piece of information that cannot be captured via expectation (but fundamentally, it seems like we ought to ensure there's an expectation capturing the observed case here -- is the relevant column all missing, e.g.?)